_build/data/transport.core.menus.php

```diff
@@ -226,10 +226,22 @@
 
 $children[1]->addMany($childrenOfClearCache, 'Children');
 
-/* Remove Locks */
+/* Trash */
 $children[2]= $xpdo->newObject('modMenu');
 $children[2]->fromArray(array (
-  'menuindex' => 2,
+    'menuindex' => 2,
+    'text' => 'trash_menu',
+    'description' => 'trash_menu_desc',
+    'parent' => 'manage',
+    'permissions' => '',
```

Comment:
It would probably be better to add some permission here, to make sure the menu is only displayed to users having the requires perm(s)

Reply:
Right. It was supposed to be "menu_trash", just forgot to add that to the build file.